### ISSUE TITLE
Can not make package management and task management to work

### ISSUE THREAD

**Issue by musahi0128** [fusioninventory/fusioninventory-for-glpi#3010]
Hi,

I have glpi 9.5.0 with FusionInventory 9.5.0+1.0.

I try to create a package like so

![image](https://user-images.githubusercontent.com/9418340/89126580-d05a6700-d510-11ea-9295-0ba738b209f7.png)

And trying to run it via task to a target like so

![image](https://user-images.githubusercontent.com/9418340/89126591-ec5e0880-d510-11ea-89c2-5587c27179cc.png)

But nothing happened.

I expect that this process will create folder 62354 on %temp% folder.

Please help.

Thank you.

**Comment by musahi0128** [comment]
@musahi0128 Anyone?

**Comment by ddurieux** [comment]
`%temp%`???

How have you installed the plugin?

**Comment by musahi0128** [comment]
@ddurieux Finally, thank you for the reply 😂

Yes i have, you can see at the first screenshoot,

Also i have install the fusioninventory-agent on the computer with full option

**Comment by ddurieux** [comment]
I mean you have downloaded and put the archive in folder glpi/plugins/?

**Comment by musahi0128** [comment]
@ddurieux I do upload a file to the package. But in the package there is command to create directory. Is it need to have any file on the package if so?

**Comment by ddurieux** [comment]
The folders are created when you install the plugin, in menu Configuration > Plugins > Fusioninventory (install button)

**Comment by musahi0128** [comment]
Oh sorry, let me rephrase,

> I expect that this process will create folder 62354 on %temp% folder.

I mean this folder should bw created on the windows client that i have install fusioninventory-agent with "Full" option during install

**Comment by ddurieux** [comment]
ah.... and in the log of agent ?

**Comment by musahi0128** [comment]
nothing about deploy module being run on the log

[fusioninventory-agent.log](https://github.com/fusioninventory/fusioninventory-for-glpi/files/5028863/fusioninventory-agent.log)

**Comment by ddurieux** [comment]
Ok, so in fact, the deploy not run on agent.
Can you set debug = 2 in registry of the agent (HKLM/Softwares/Fusioninventory-agent), restart the agent service and run again the agent.
We will have more information in the agent log

**Comment by musahi0128** [comment]
@ddurieux The log i send is already on debug level 2

**Comment by ddurieux** [comment]
Ok, try http://127.0.0.1:62354/now to force agent execution, will see more details.

Wait, the server url must finish with the `/` like `/fusioninventory/`

**Comment by musahi0128** [comment]
> Wait, the server url must finish with the / like /fusioninventory/
Done

Here's the log file

[fusioninventory-agent.log](https://github.com/fusioninventory/fusioninventory-for-glpi/files/5029082/fusioninventory-agent.log)

**Comment by ddurieux** [comment]
Seems the URL for deploy is wrong (http://192.168.22.11). Go in Administration > Entity > Root entity and in tab _Fusioninventory_ update the field _Service URL_. With the log, seems to be _http://192.168.22.249/glpi_

**Comment by musahi0128** [comment]
![AnyDesk_Aif9PnpNHW](https://user-images.githubusercontent.com/9418340/89429556-e746c680-d767-11ea-9b3b-d56f8a23571b.png)
![chrome_OwTIAfHcLm](https://user-images.githubusercontent.com/9418340/89429563-e877f380-d767-11ea-8734-3e665ac94c40.png)

Finally!!! thanks dude, god bless you :-*

**Comment by ddurieux** [comment]
Nice ;)
